installation-usage.md

```diff
@@ -137,6 +138,39 @@ spec:
     - exit_group
 ```
 
+#### Bind workloads to profiles with ProfileBindings
```

Comment:
Do we need to document the cert-manager dependency?

Reply:
I think that would be good.

Reply:
Let's also document what exactly happens when cert manager does not exist :+1:

Reply:
Oops, good catch. Added now as part of the installation instructions.

Currently the webhook is bundled into the regular operator deployment, so failing to install cert-manager will result in the operator.yaml failing to deploy. We had talked about splitting the webhook into a separate installation, do you think we should do that so that cert-manager isn't always required?

Reply:
> Currently the webhook is bundled into the regular operator deployment, so failing to install cert-manager will result in the operator.yaml failing to deploy. We had talked about splitting the webhook into a separate installation, do you think we should do that so that cert-manager isn't always required?

I would be in favor if the operator still works either with or without cert-manager to be available. Maybe we can change the controller init logic that it will only warn if cert-manager is not available (aka the cert is not available) and then disable the feature automatically?

I would not split it into a separate deployment variant, but we could also think about making it manually configurable via the configmap.

Let's follow-up on that topic after this PR got merged. WDYT?